apperals-frontend/src/App.js

```diff
@@ -38,6 +41,9 @@ function App() {
           <Route path="/" element={<ButtonRow />} />
```

Comment:
Need to create another component which would hold the Routes Array.
The intended purpose for App.js is to render the structure for the landing page.